docs/reference/serializers.rst

```diff
@@ -16,6 +16,22 @@ format, namely JSON. This allows us to use the same business logic for the serve
 the client. It also means, that we could create native mobile apps, which communicate with a
 web-application, without ever seeing a line of HTML code.
 
+Moreover, since **django-SHOP** uses **django-CMS** to organize all available components, a classic
+Django "View" does not make much sense anymore. Therefore, as we evolve our Model-View-Control
+pattern into a modern web application, our REST serializers become the new controllers.
+
```

Comment:
Not sure I agree with that (see my general comment). Technically, we still have a Django view, and the serializer's job is to perform serialization / deserialization. But I also think our documentation should not get too much into philosophical discussions of MVC :)